firmware/lib/Gantry/Timer.h

```diff
@@ -1,14 +1,24 @@
 #ifndef TIMER_H
 #define TIMER_H
 
-#include "Arduino.h"
+#include <Arduino.h>
 
-void start_timer(Tc *tc, uint32_t channel, IRQn_Type irq, uint32_t velocity);
+#define TC_IRQN_I(_x)          TC##_x##_IRQn
+/** Generates the IRQn value corresponding to a TC IRQ number */
+#define TC_IRQN(_x)            TC_IRQN_I(_x)
 
-void reset_timer(Tc *tc, uint32_t channel, IRQn_Type irq, uint32_t velocity);
+#define TC_ISR_I(_x)           void TC##_x##_Handler()
+/** Generates the ISR function name corresponding to a TC IRQ number */
+#define TC_ISR(_x)             TC_ISR_I(_x)
 
-void start_timer_accel(Tc *tc, uint32_t channel, IRQn_Type irq, uint32_t accel);
+// PWM Timers
+void configure_pwm_timer(Tc *tc, uint32_t channel, IRQn_Type irq, Pio *pio_bank, EPioType periph, uint32_t pin_mask);
```

Comment:
Just realizing that all the functions use underscore convention as opposed to camelCase?

Reply:
Most of the C code I've ever dealt with uses `snake_case`.
Idk what the C++ convention is.